# file: boxbrainiac/error.py
error_message = {
    'MAI-000': 'An Unknown error occurred in main.py: ',
    'STO-001': 'Can not read file {} or load the YAML data in store.py: ',
    'STO-002': 'Can not write file {} )or dump YAML data in store.py: ',
    'GIT-004': 'Can not git pull in git.py: ',
    'GIT-005': 'Can not git commit and push in git.py: ',
    'DAT-006': 'An error occurred while processing the data in manage_boxes: ',
    'DAT-007': 'An error occurred while processing the data in edit_view GET: ',
    'DAT-008': 'An error occurred while processing the data in edit_view POST: ',
    'DAT-009': 'An error occurred while processing the data in delete_box_view: ',
    'DAT-010': 'An error occurred while processing the data in list_view: ',
    'DAT-011': 'An error occurred while processing the data in search_view: ',
    }
